adc/onboard_temperature/onboard_temperature.c

```diff
@@ -17,7 +17,7 @@
  * pico-examples/adc/adc_console/adc_console.c */
 float read_onboard_temperature(const char unit) {
     
-    /* 12-bit conversion, assume max value == ADC_VREF == 3.3 V */
+    /* 12-bit conversion, assume max value == ADC_VREF == 3.3 V */4096
```

Comment:
That looks like a compile error

Reply:
Yeah, did you mean to put the `4096` **inside** the comment @ashleey30 ?